.github/workflows/gemini-review.yml

```
@@ -0,0 +1,130 @@
+          DIFF: ${{ steps.diff.outputs.diff }}
+        run: |
+          if [ "${{ steps.diff.outputs.truncated }}" = "true" ]; then
+            echo "verdict=SKIP" >> $GITHUB_OUTPUT
+            echo "Diff truncated, skipping Gemini review"
+            exit 0
+          fi
+
+          PROMPT="あなたは個人の自己紹介サイト（HTML/CSS静的サイト）のコードレビュアーです。以下のPR diffをレビューし、JSONのみで回答してください。
+
+## 【最重要】個人情報・機密情報の漏洩チェック
+以下が含まれている場合は必ずCOMMENTとしてください：
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)